alacritty_terminal/src/config/bindings.rs

```diff
@@ -71,113 +71,158 @@ pub fn default_key_bindings() -> Vec<KeyBinding> {
     let mut bindings = bindings!(
         KeyBinding;
         Key::Paste; Action::Paste;
-        Key::Copy; Action::Copy;
+        Key::Copy;  Action::Copy;
```

Comment:
I don't think there's any reason for this whitespace change?

Reply:
I had the same comment, but it's to align with the above action. This is consistent with the rest of this `bindings!` macro invocation:

https://github.com/jwilm/alacritty/blob/748138240508f9a032cfde9f0e397f7fc3be5eb8/alacritty_terminal/src/config/bindings.rs#L89-L90

Reply:
Oh right. Somehow I checked the action above it and came to the conclusion they do not line up. But very clearly they do.

I think I even noticed this in the last review and didn't comment because I saw that they line up. So please just disregard.